### ISSUE TITLE
Fix indentation in index.html page of new projects.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1661]
The ``head`` and ``body`` tags of the index.html page should be indented one extra level. Right now they are flush with the ``html`` tag, which is incorrect.

![image](https://cloud.githubusercontent.com/assets/25212/22259691/25c972a4-e21b-11e6-926e-412974fa323a.png)

**Comment by flukeout** [comment]
I think in general our spacing is a bit clunky, maybe we can just go with a more compact layout like this.

```
<!DOCTYPE html>
<html>
  <head>
    <title>Made with Thimble</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="style.css">
  </head>
  <body>
    <h1>Welcome to Thimble</h1>
    <p>Make something <strong>amazing</strong> with the web!</p>
  </body>
</html>
```

This also moves to the ``title`` tag to the first spot in the ``head`` which is good. Additionally, we could include a couple of common element types on this page, like an ``a`` and an ``ol``, we could use it to link to some resources. Any good HTML or CSS starter guides?

**Comment by Rajat-dhyani** [comment]
hello sir,
i would like to fix this issue. can you please locate me the file?
Thank you in advance

**Comment by humphd** [comment]
Fixing this requires editing this file to reflect the changes @flukeout wants above:

https://github.com/mozilla/thimble.mozilla.org/blob/master/public/resources/default-files/html.txt

**Comment by Rajat-dhyani** [comment]
okay great,
thank you sir :)
Did the necessary changes :D

**Comment by flukeout** [comment]
Okay, thanks so much @Rajat-dhyani - we've merged your PR.

**Comment by gideonthomas** [comment]
Re-opening due to: https://github.com/mozilla/thimble.mozilla.org/pull/1663#issuecomment-276567176

**Comment by Rajat-dhyani** [comment]
@gideonthomas  @flukeout please check this pr :
https://github.com/mozilla/thimble.mozilla.org/pull/1689

**Comment by gideonthomas** [comment]
Closed with 4f1fa5d120c94277e68e34d4a23b0223046b0d56 and f25a85681cee5391ad248c7b32c5e6d70efc9383
